prereise/gather/demanddata/transportation_electrification/daily_trip_charging.py

```diff
@@ -0,0 +1,190 @@
+import math
+
+import numpy as np
+from scipy.optimize import linprog
+
+from prereise.gather.demanddata.transportation_electrification import (
+    charging_optimization,
+    dwelling,
+)
+
+
+def calculate_daily_smart_charging_trips(
+    newdata,
+    input_day,
+    day_iter,
+    data_day,
+    adjusted_load,
+    charging_efficiency,
+    daily_vmt_total,
+    kwh,
+    bev_vmt,
+):
```

Comment:
I'm still adding doc strings to this function and might remove a couple parameters